<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WB_PaintWindow</class>
 <widget class="QMainWindow" name="WB_PaintWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>735</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="WB_GraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarCommon">
   <property name="windowTitle">
    <string>Common</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBarZoom">
   <property name="windowTitle">
    <string>Zoom</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomOriginal"/>
   <addaction name="actionZoomFit"/>
   <addaction name="separator"/>
   <addaction name="actionGrid"/>
  </widget>
  <widget class="QToolBar" name="toolBarTools">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectMove"/>
   <addaction name="actionPen"/>
   <addaction name="actionHighlight"/>
   <addaction name="actionText"/>
   <addaction name="actionPointer"/>
   <addaction name="actionPanZoomTool"/>
  </widget>
  <widget class="QToolBar" name="toolBarSettings">
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarEdit">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSelectAll"/>
   <addaction name="actionClear"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QToolBar" name="toolBarFont">
   <property name="windowTitle">
    <string>Font</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/text_fields_black_24dp.svg</normaloff>:/icons/icons/text_fields_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionHighlight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/drive_file_rename_outline_black_24dp.svg</normaloff>:/icons/icons/drive_file_rename_outline_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
   <property name="toolTip">
    <string>Highlight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/content_paste_black_24dp.svg</normaloff>:/icons/icons/content_paste_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/select_all_black_24dp.svg</normaloff>:/icons/icons/select_all_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/content_copy_black_24dp.svg</normaloff>:/icons/icons/content_copy_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/file_download_black_24dp.svg</normaloff>:/icons/icons/file_download_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/file_upload_black_24dp.svg</normaloff>:/icons/icons/file_upload_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/delete_outline_black_24dp.svg</normaloff>:/icons/icons/delete_outline_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionZoomOriginal">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/zoom_original_black_24dp.svg</normaloff>:/icons/icons/zoom_original_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Original</string>
   </property>
   <property name="toolTip">
    <string>Zoom Original</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/crop_free_black_24dp.svg</normaloff>:/icons/icons/crop_free_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom Fit</string>
   </property>
  </action>
  <action name="actionPointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/radio_button_checked_black_24dp.svg</normaloff>:/icons/icons/radio_button_checked_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Pointer</string>
   </property>
   <property name="toolTip">
    <string>Pointer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/create_black_24dp.svg</normaloff>:/icons/icons/create_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="toolTip">
    <string>Pen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionPanZoomTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/pan_tool_black_24dp.svg</normaloff>:/icons/icons/pan_tool_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Pan and Zoom Tool</string>
   </property>
   <property name="toolTip">
    <string>Pan and Zoom Tool (Ctrl for Zoom)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionSelectMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/select_black_24dp.svg</normaloff>:/icons/icons/select_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Select/Move</string>
   </property>
   <property name="toolTip">
    <string>Select or Move</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/content_cut_black_24dp.svg</normaloff>:/icons/icons/content_cut_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/grid_4x4_black_24dp.svg</normaloff>:/icons/icons/grid_4x4_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle Grid</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/export_black_24dp.svg</normaloff>:/icons/icons/export_black_24dp.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export to File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WB_GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>views/wb_graphics_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
